scripts/ci/ci_run_airflow_testing.sh

```diff
@@ -71,6 +71,11 @@ INTEGRATIONS=()
 
 ENABLED_INTEGRATIONS=${ENABLED_INTEGRATIONS:=""}
 
+if [[ ${TEST_TYPE:=} == "Integration" ]]; then
+    export ENABLED_INTEGRATIONS="${AVAILABLE_INTEGRATIONS}"
+    export RUN_INTEGRATION_TESTS="${AVAILABLE_INTEGRATIONS}"
```

Comment:
Again just a query, should we just use `AVAILABLE_INTEGRATIONS` ? It feels like 3 variables that are closely related i.e. is there a time when `RUN_INTEGRATION_TESTS != ENABLED_INTEGRATIONS` ?

Reply:
Not really - because AVAILABLE_INTEGRATIONS is now it's of all that are available and it is now checked if it the same as in "breeze complete" (there must be a different list because it hast be "standalone" as it is sourced in ~/bash_complete.d/ - so ./breeze_complete script cannot source anything else (like _common)

ENABLED_INTEGRATIONS on the other hand are just telling which integration are enabled for this particular test. And currently it looks like they all can work in the same single operation,

RUN_INTEGRATIONS tells which integration tests are executed and it means that ONLY those integration tests are executed. I can rename it later to "ONLY_RUN_INTEGRATIONS" to make it clearer.


There is one case that I might want to enable:

ENABLED_INTEGRATIONS="kerberos"
RUN_INTEGRATIONS=""

I have forgotten about this case and I will add it shortly - possibly when I integrate Kubernetes tests in Github actions. Airflow behaves a bit differently with/without kerberos variables (it auto-detects if kerberos is enabled)  so there should be at least one pass where we want to run Airflow with Kerberos but run all non-kerberos-integration specific tests.

I will add it as separate change. Good that you asked :)